pom.xml

```diff
@@ -357,6 +357,11 @@
     <!-- dependencies are stored in the relevant submodules:
           see main/pom.xml, server/pom.xml and each extension.
     -->
+      <dependency>
+        <groupId>org.json</groupId>
```

Comment:
We cannot add this dependency to the project because it has an incompatible license. Jackson (which we already depend on) should have all the necessary features for our JSON processing needs. Also, it does not look like you are using this new dependency in your changes at all.

Reply:
You're right, I will remove it. I was sloppy leaving it there :c